testdata/project-v2/Makefile

```diff
@@ -52,7 +48,7 @@ vet:
 
 # Generate code
 generate: controller-gen
-	$(CONTROLLER_GEN) object:headerFile=./hack/boilerplate.go.txt paths="./..."
+	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
```

Comment:
I think this may need a rebase as #1287 was merged including this change

Reply:
rebased with master